packages/flutter_test/lib/src/binding.dart

```diff
@@ -1179,6 +1179,8 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     RendererBinding.instance.initMouseTracker();
     // ignore: invalid_use_of_visible_for_testing_member
     ServicesBinding.instance.resetInternalState();
```

Comment:
a generic reset method like this would be much closer to what would be acceptable for resetting/disposing debugShowWidgetInspectorOverride. You definitely don't want to pollute the important WidgetBindings class with a method that specifically disposed the inspector override. What happens when someone adds a second disposable thing to the bindings? Similarly there might be other disposable things on the binding that we are just missing.